webbeans-jetty9/pom.xml

```diff
@@ -129,30 +129,102 @@
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-assembly-plugin</artifactId>
+                <!-- TODO switch back to jetty descriptors once https://github.com/eclipse/jetty.project/issues/3514
+                       is resolved -->
+                <!--<dependencies>-->
+                <!--    <dependency>-->
+                <!--        <groupId>org.eclipse.jetty.toolchain</groupId>-->
+                <!--        <artifactId>jetty-assembly-descriptors</artifactId>-->
+                <!--        <version>1.0</version>-->
+                <!--    </dependency>-->
+                <!--</dependencies>-->
+                <executions>
+                    <execution>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>single</goal>
+                        </goals>
+                        <configuration>
+                            <!--<descriptorRefs>-->
+                            <!--    <descriptorRef>config</descriptorRef>-->
+                            <!--</descriptorRefs>-->
+                            <descriptors>
+                                <descriptor>src/main/assembly/config.xml</descriptor>
+                            </descriptors>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <!-- NOTE may need to switch to a different plugin if
+                       https://github.com/eclipse/jetty.project/issues/3514 provides a solution for transitive
+                       dependencies -->
+                <artifactId>maven-resources-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>copy-resources</id>
+                        <phase>prepare-package</phase>
+                        <goals>
+                            <goal>copy-resources</goal>
+                        </goals>
+                        <configuration>
+                            <outputDirectory>${project.build.directory}/config</outputDirectory>
+                            <resources>
+                                <resource>
+                                    <directory>src/main/config</directory>
+                                    <filtering>true</filtering>
+                                </resource>
+                            </resources>
+                            <useDefaultDelimiters>false</useDefaultDelimiters>
+                            <delimiters>
+                                <delimiter>@</delimiter>
+                            </delimiters>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>mrm-maven-plugin</artifactId>
```

Comment:
need to use mrm because `jetty.jar --add-to-start` needs a http url to download from and we want to use the just-built artifacts